### PR TITLE
Display osmo apr

### PR DESCRIPTION
**Pull request by fabryscript**

### CODE REVIEW

File: components/sections/stake-section/index.tsx [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
-          <h4 className="bg-stake-heading -tracking-2% text-center font-poppins text-3.5xl font-medium leading-9.5 md:text-4xl md:leading-10.75 lg:text-5xl lg:leading-13 xl:text-5.5xl xl:leading-15.25 2xl:text-6.25xl 2xl:leading-17.5">
+          <h4 className="bg-stake-heading text-center font-poppins text-3.5xl font-medium leading-9.5 -tracking-2% md:text-4xl md:leading-10.75 lg:text-5xl lg:leading-13 xl:text-5.5xl xl:leading-15.25 2xl:text-6.25xl 2xl:leading-17.5">
             Stake to earn <br className="sm:hidden" />
             rewards.
           </h4>
           <p className="self-stretch text-center font-light leading-5.5 text-neutral-100 lg:text-xl lg:leading-7">
             Lock up your OSMO and earn passive <br className="sm:hidden" />
             staking <br className="hidden sm:block" /> rewards with up to{" "}
-            <span className="text-malachite-200">10.86% APR.</span>
+            <span className="text-malachite-200">{stakingApr} APR.</span>
```
Comment (DavideSegullo) [inline components/sections/stake-section/index.tsx:28]: Don't do server component call inside the whole section:

```
const stakingApr = new RatePretty((await queryOsmoAPR())[0].apr / 100)
    .maxDecimals(2)
    .toString();
```

But extract this code in a different server component, and here you can add a suspense with a skeleton:
```
<span className="text-malachite-200">{stakingApr} APR.</span>
```

**Review by DavideSegullo — APPROVED** [review]

File: components/sections/stake-section/index.tsx (hunk 1 above)
Comment (fabryscript) [reply components/sections/stake-section/index.tsx]: [4a52668](https://github.com/osmosis-labs/osmosis-website/pull/63/commits/4a52668d4de4eab4a922d3357623b824e3a341c9)

File: components/sections/stake-section/index.tsx [hunk 2]
```diff
@@ -1,25 +1,35 @@
...
 import Image, { getImageProps } from "next/image";
+import { Suspense } from "react";
 
 const airdrops = Array<{ uri: string; name: string }>(22).fill({
   uri: "/assets/icons/pepe.svg",
   name: "Pepe",
 });
 
-export default function StakeSection() {
+export default async function StakeSection() {
```
Comment (DavideSegullo) [inline components/sections/stake-section/index.tsx:11]: We don't need the `async` keyword

File: components/sections/stake-section/index.tsx [hunk 3]
```diff
@@ -1,25 +1,35 @@
...
+          <h4 className="bg-stake-heading text-center font-poppins text-3.5xl font-medium leading-9.5 -tracking-2% md:text-4xl md:leading-10.75 lg:text-5xl lg:leading-13 xl:text-5.5xl xl:leading-15.25 2xl:text-6.25xl 2xl:leading-17.5">
             Stake to earn <br className="sm:hidden" />
             rewards.
           </h4>
           <p className="self-stretch text-center font-light leading-5.5 text-neutral-100 lg:text-xl lg:leading-7">
             Lock up your OSMO and earn passive <br className="sm:hidden" />
             staking <br className="hidden sm:block" /> rewards with up to{" "}
-            <span className="text-malachite-200">10.86% APR.</span>
+            <Suspense
+              fallback={
```
Comment (DavideSegullo) [inline components/sections/stake-section/index.tsx:25]: Can we use an empty string for 10.88%, or invalid text may be indexed

File: components/sections/stake-section/staking-apr.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { queryOsmoAPR } from "@/lib/queries/numia";
+import { RatePretty } from "@keplr-wallet/unit";
+import React from "react";
+
+export default async function StakingApr() {
+  // await new Promise<void>((res) => setTimeout(() => res(), 1000 * 2));
```
Comment (DavideSegullo) [inline components/sections/stake-section/staking-apr.tsx:6]: Do we need this row? Can we delete it?

**Review by DavideSegullo — CHANGES_REQUESTED** [review]

**Review by DavideSegullo — APPROVED** [review]

**Pull request merged**